lib/pure/unittest.nim

```diff
@@ -20,14 +20,24 @@
 ## Compiled test files return the number of failed test as exit code, while
 ## ``nim c -r <testfile.nim>`` exits with 0 or 1
 ##
-## Running a single test
-## ---------------------
+## Selecting tests to run
```

Comment:
Please add your new globbing example separately (under a different heading). Most people will be searching for how to run a single test, they won't want to scan through a globbing example.

Reply:
I would actually add an additional header named "Running a single suite"